src/mlpack/tests/lsh_test.cpp

```diff
@@ -917,4 +917,74 @@ BOOST_AUTO_TEST_CASE(MoveOperatorTest)
   CheckMatrices(distances, distances2);
 }
 
+/**
+ * Run LSH on (identical) dense and sparse data, making sure that we get the
+ * same results.  Note that the sparse data we are using isn't really
+ * sparse---the idea is to test that the class works correctly when the type is
+ * arma::sp_mat.
+ */
+BOOST_AUTO_TEST_CASE(SparseLSHTest)
+{
+  mlpack::math::RandomSeed(std::time(NULL));
```

Comment:
Looks like we can remove the line, probably a left over of the testing process.

Reply:
Thanks, nice catch!  Fixed in 883c0fc.